docs/topics_5/web-running-saved-analysis.adoc

```diff
@@ -0,0 +1,16 @@
+
+
+// Module included in the following assemblies:
+// * docs/web-console-guide_5/master.adoc
+[id='running_saved_analysis_{context}']
+= Running a saved analysis
+
+You can run the last saved version of a project from the *Analysis results* screen.
```

Comment:
I don't think the user is running a "version of a project". More likely to be the last saved version of an analysis, since that is the title.

Reply:
Now that I've stepped back a bit, I think the section should be retitled "Analyzing a saved project.' This is the case where you previous clicked *Save*, not *Save and run*, in either the *Create project* wizard or the *Analysis configuration* screen. You are analysing a project that you saved without running an analysis. WDYT?

Reply:
It's still a saved analysis that you are running. "Analyzing a saved project" is not quite right. 

All projects are saved. So the "saved project" part does not make sense. 

I think the current title is fine. Don't make it more complicated.